tutorials/fundamentals/check-nans.md

```diff
@@ -0,0 +1,11 @@
+When dealing with NaNs in your projects, it is important to understand what `NaNs` are and how they work. `NaN` is a non-writable, non-configurable, non-enumerable property of the global object. It is of type number and insted of overriding it, you should error handle it when the situation arises in your project development. To do that however, you must check whether the value is a `NaN`. An interesting thing about `NaNs` is that `NaN !== NaN` and `Number.NaN !== NaN`. You should avoid using `isNan()` as it will check if the given value would result in a `NaN` if you tried to convert it to a number. Instead you can do the following:
```

Comment:
Also, instead of saying "avoid using `isNan()`", you should recommend using `Number.isNaN()`. Maybe a separate paragraph below on why you shouldn't use `isNaN()` using the examples from here: https://masteringjs.io/tutorials/fundamentals/nan#checking-for-nan